src/scss/pages.scss

```diff
@@ -123,6 +123,62 @@
     }
 }
 
+#sso-page {
+    .content {
+        width: 300px;
+
+        .box {
+            margin-top: 30px;
+            margin-bottom: 30px;
+            text-align: center;
+        }
```

Comment:
I dont see a "box" in your screenshot, so you shouldnt need to use this style.

Reply:
It's used on the `sso.component.html` page.